witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -13,7 +13,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-var witHtmlLocation = require('file-loader!./wit_jupyter.html');
```

Comment:
What did this require call default to? With the new changes this will default to	  '/nbextensions/wit-widget/wit_jupyter.html' per line L17 if I understood this correctly. Previously it defaulted to this require call's return due to null check in L56.

Reply:
not sure of previous exact behavior but tested this change in all scenarios and works across the board (also matches TFMA's widget code now)